pkg/pki/ca/ca.go

```diff
@@ -22,10 +22,24 @@ import (
 	"errors"
 	"time"
 
+	"github.com/golang/glog"
 	"istio.io/auth/pkg/pki"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
 )
 
 const (
+	// istioSecretType is the Istio secret annotation type.
+	istioSecretType = "istio.io/key-and-cert"
```

Comment:
'key-and-cert- is too generic - maybe 'ca-root' ?

Reply:
done.